pom.xml

```diff
@@ -91,6 +91,12 @@
                 <artifactId>slf4j-api</artifactId>
                 <version>${org.slf4j.verison}</version>
             </dependency>
+            <!-- https://mvnrepository.com/artifact/org.wso2.orbit.com.nimbusds/nimbus-jose-jwt -->
```

Comment:
We can remove this comment

Reply:
Fixed 6646410ccf9035a93f8e161a8c67baed03d9b73e